sites/next.skeleton.dev/src/examples/resources/cookbook/image-layouts/ExampleGrid.astro

```diff
@@ -0,0 +1,36 @@
+<!-- Flex -->
+<section class="flex justify-center">
+	<!-- Grid -->
+	<div class="grid grid-cols-2 gap-6 md:grid-cols-3">
+		<!-- Row -->
+		<div class="h-48 w-48">
+			<img class="h-full w-full rounded-lg object-cover" src="https://images.unsplash.com/photo-1617296538902-887900d9b592" alt="" />
+		</div>
+		<div class="h-48 w-48">
+			<img class="h-full w-full rounded-lg object-cover" src="https://images.unsplash.com/photo-1597077962467-be16edcc6a43" alt="" />
```

Comment:
Let's make sure we're populating `alt` tags to follow best practices for accessibility.